pkg/KubeArmorOperator/internal/controller/cluster.go

```diff
@@ -1284,35 +1284,43 @@ func UpdateRecommendedPolicyConfig(config *opv1.KubeArmorConfigSpec) bool {
 
 func UpdateConfigMapData(config *opv1.KubeArmorConfigSpec) bool {
 	updated := false
+	configMapData := ""
```

Comment:
we can encode common.ConfigMapData itself to valid yaml data right! we don't need to create this formatted string. WDYT?